packages/api/src/routes/lightclient.ts

```diff
@@ -57,16 +59,13 @@ export function getReqSerializers(): ReqSerializers<Api, ReqTypes> {
       schema: {params: {stateId: Schema.StringRequired}, body: Schema.AnyArray},
     },
 
-    getBestUpdates: {
+    getCommitteeUpdates: {
       writeReq: (from, to) => ({query: {from, to}}),
       parseReq: ({query}) => [query.from, query.to],
       schema: {query: {from: Schema.UintRequired, to: Schema.UintRequired}},
     },
 
-    getLatestUpdateFinalized: reqEmpty,
-    getLatestUpdateNonFinalized: reqEmpty,
-
-    getInitProof: {
+    getSnapshot: {
```

Comment:
Cool, this new format is way more compact than previously. Furthermore, a client who just wants to sync and is not interested in the state-proof functionality does not require multiproof functionality anymore.